### DISCUSSION TITLE
Welcome to our-handbook Discussions for the E-life Innovation Sprint!

### DISCUSSION THREAD

**Discussion by NatalieZelenka** [very-good-science/our-handbook-template#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

 To get started, comment below with an introduction of yourself and tell us about what you'd like to get out of participating in the sprint!

**Comment by NatalieZelenka** [comment]
Hi, I'm Natalie Thurlby, I'm a Data Scientist at the University of Bristol 👋 . During this sprint I'm looking forward to getting to know you all and including as many of your great ideas as I can. I also hope that I'll be able to offer any support that you might want/need.

**Reply by ekuw** [reply]
Hello  👋 I'm Emma Kuwertz, also a Data Scientist at the University of Bristol. Looking forward to meeting everyone!
